neuro_evolution_ctrnn/optimizer/i_optimizer.py

```diff
@@ -19,8 +19,7 @@ class IOptimizer(abc.ABC, Generic[ConfigClass]):
 
     @abc.abstractmethod
     def __init__(self, eval_fitness: Callable, individual_size: int, random_seed: int, conf: ConfigClass, stats,
-                 map_func=map,
-                 from_checkoint=None):
+                 map_func=map, from_checkoint=None, reset_hof=False):
```

Comment:
Müssen wir den Parameter `reset_hof` überhaupt hier in den `IOptimizer` geben? In CMA und Mu+Lambda wird der gar nicht nach oben weitergeleitet und nur in deren respektiven Konstruktoren direkt genutzt